etc/config/odin.amazon.properties

```diff
@@ -0,0 +1,13 @@
+compilers=&odin
+defaultCompiler=odin2209
+group.odin.compilers=odin2209:odin2208
+group.odin.objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
+group.odin.demangler=
+group.odin.isSemVer=false
+group.odin.baseName=odin
+group.odin.compilerType=odin
+group.odin.versionFlag=version
```

Comment:
As of a few minutes ago, we're asking compiler groups to provide license info
`group.odin.licensePreamble` for custom text to be shown,
`group.odin.licenseName` for the license name,
`group.odin.licenseLink` for either a link to the license or other relevant URL.

Can you provide those? All three are optional if you don't have anything for the preamble, for example